packages/twenty-front/src/modules/settings/data-model/fields/forms/number/components/SettingsDataModelFieldNumberForm.tsx

```diff
@@ -69,7 +69,7 @@ export const SettingsDataModelFieldNumberForm = ({
             <SettingsOptionCardContentCounter
               Icon={IconDecimal}
               title="Number of decimals"
-              description={`Example: ${(1000).toFixed(count)}`}
+              description={`Example: ${(1000).toFixed(count)} ${type === 'percentage' ? '%' : ''}`}
```

Comment:
```suggestion
              description={`Example: ${(type === 'percentage' ? 99 : 1000).toFixed(count)} ${type === 'percentage' ? '%' : ''}`}
```

Reply:
To match design
![image](https://github.com/user-attachments/assets/594e272a-1020-4d9a-ad91-69c2e7b00eaa)